anvil_consortium_manager/models.py

```diff
@@ -79,6 +82,30 @@ def anvil_import(cls, billing_project_name):
         billing_project.save()
         return billing_project
 
+    @classmethod
+    def anvil_audit(cls):
```

Comment:
Consider splitting this into two methods in general:
1. check if all records in the app exist in AnVIL - this can return a list of objects that are missing on AnVIL.
2. check if AnVIL has any records that don't exist in the app - this can return ???

Then the `anvil_audit()` method could call both of those methods and raise an exception if there were any issues?

Reply:
I ended up not splitting into sub functions to minimize the number of API calls. Additionally, some of the checks that are done are dependent on previous checks (eg, if a group exists in the app but not on AnVIL, it does not make sense to check the group membership on AnVIL).